pkg/disttask/framework/scheduler/scheduler_manager.go

```diff
@@ -120,9 +121,18 @@ func (sm *Manager) Start() {
 	failpoint.Inject("disableSchedulerManager", func() {
 		failpoint.Return()
 	})
+	// init cached managed nodes
+	sm.nodeMgr.refreshManagedNodes(sm.ctx, sm.taskMgr)
```

Comment:
what if refresh failed?

Reply:
it's called periodicly per second, here just init it to make test pass